PythonicPenguins/bangazon_api
# Move gitignore to root

## Status
READY

## Migrations
NO

## Description
Move `.gitignore` file to the root directory and modify the migrations path in the `.gitignore`.

## Related PRs
N/A

## Resolves Issue Number
N/A

## Deploy Notes
run `git checkout master` and then `git pull`.

## Steps to Test or Reproduce
Make changes or delete any of the migrations files. Go to terminal and run `git status` and make sure that migrations folder is not showing up as a folder to add.

## Impacted Areas in Application
List general components of the application that this PR will affect:
N/A
